util.go

```diff
@@ -40,7 +40,7 @@ func init() {
 	}
 }
 
-func validateAndPopulateAuroraURL(urlStr string) (string, error) {
+func validateAuroraURL(urlStr string) (string, error) {
```

Comment:
The argument name does not have to indicate its type, it is right there next to it

Reply:
That's right, thanks for catching this.